runtime/parachains/src/scheduler.rs

```diff
@@ -59,7 +59,7 @@ pub use pallet::*;
 #[cfg(test)]
 mod tests;
 
-const LOG_TARGET: &str = "runtime::parachain_scheduler";
+const LOG_TARGET: &str = "runtime::parachains::scheduler";
```

Comment:
Why was the log target changed?

Reply:
The log target isn't set in the scheduler prior to the scheduling changes. This brings it in line with what I am using in `assigner_on_demand` as well as the `slashing` module.